README.md

```diff
@@ -1,232 +1,110 @@
-the respective project page.
+## Building a BISDN Linux image
 
-## Bootstrap build system
+### x86_64 platforms
 
-The commands needed to bootstrap and initialise your build system are shown
-below. Before starting to copy paste those, please make sure to set $BRANCHNAME
-in your environment to an existing branch in the bisdn/bisdn-linux repo from
-which you want to start your build (e.g. `export BRANCHNAME=v4.8.0` or `export
-BRANCHNAME=master`)
+The default image works on all x86_64 platforms supported by BISDN
+Linux. To build the image from the latest sources, clone this repo and
+run the following command in its root directory:
 
 ```bash
-# init repo
-mkdir -p ~/workspace/poky-bisdn-linux
-cd !$
-repo init -b ${BRANCHNAME} -u https://github.com/bisdn/bisdn-linux.git
-
-# sync repos
-repo sync
-
-# init build system
-TEMPLATECONF="meta-switch/conf/templates/bisdn-linux" . poky/oe-init-build-env poky/build-bisdn-linux/
+kas-container build bisdn-linux.yaml
 ```
 
-## Configure target machine and cache directory
+After the build process has finished, your image will be in
+`build/tmp/deploy/images/generic-x86-64/`. A symlink named
+`onie-bisdn-full-generic-x86-64.bin` will point to the actual image
+file named `onie-bisdn-full-generic-x86-64-<timestamp>.bin`.
 
-Edit conf/local.conf and set `MACHINE` to your desired target. Cached files and built artifacts
-will be stored in `/tmp` by default. Change the variables `SSTATE_DIR`,
-`TMPDIR`, `DL_DIR` if you wish to use a different location. See the
-[Yocto Project Reference Manual: Variables Glossary](https://docs.yoctoproject.org/ref-manual/variables.html)
-for a complete overview of variable definitions.
+### ARM platforms (Edgecore AS4610)
 
-## Build image
+To build an image that works on all ARM platforms supported by BISDN Linux,
+run this command in the root of this repository:
 
-Chose one of the available image types to build
+```bash
+MACHINE=generic-armel-iproc kas-container build bisdn-linux.yaml
+```
 
-* `minimal`: bare minimum of packages for booting the system. Does not include any
-  closed source packages outside of required firmware files.
+After the build process finishes, your image will be in
+`build/tmp/deploy/images/generic-armel-iproc/`. A symlink named
+`onie-bisdn-full-generic-armel-iproc.bin` will point to the actual image
+file named `onie-bisdn-full-generic-armel-iproc-<timestamp>.bin`.
 
-* `full`: include the full BISDN Linux system, including baseboxd and OF-DPA.
+### ofdpa-gitlab
 
-> **_NOTE:_** When trying to build a release prior to 4.6.1, please read the
-> [Known issues when building older versions](#known-issues-when-building-older-versions) section before starting.
+If you want to build ofdpa from source and you have access to the ofdpa-gitlab
+repo, the most convenient way is to use your `.ssh` directory (to avoid
+complaints that the authenticity of the host can't be established) and your
+ssh-agent (for authentication with gitlab.bisdn.de):
```

Comment:
```suggestion
If you want to build ofdpa from source, you need access to the ofdpa-gitlab
repo. The most convenient way to authenticate, while building inside the kas-container,
is to use your own `.ssh` directory and ssh-agent (to avoid
complaints that the authenticity of the host can't be established), by passing the command line
arguments as shown below:
```

Reply:
I kept some of the original phrasing (the information in parentheses).